src/jshint.js

```diff
@@ -1419,12 +1425,23 @@ var JSHINT = (function() {
     var x = symbol(s, 150);
 
     x.led = function(left) {
+      // this = suffix e.g. "++" punctuator
+      // left = symbol operated e.g. "a" identifier or "a.b" punctuator
+
       if (state.option.plusplus) {
         warning("W016", this, this.id);
       } else if ((!left.identifier || isReserved(left)) && left.id !== "." && left.id !== "[") {
         warning("W017", this);
       }
 
+      // detect modification of a const.
+      // in the case of a.b, left will be the "." punctuator
```

Comment:
> You deleted the comment on github (or it otherwise vanished for some other reason), so I wasn’t able to reply on github and can’t see what you’re talking about.

when a line is updated in a pr, github hides the comment.

You can see the comments by finding the "caitp commented on an outdated diff" line and click "Show outdated diff".